reference/configuration/framework.rst

```diff
@@ -3596,13 +3595,9 @@ exceptions that match the given exception class:
         use Symfony\Config\FrameworkConfig;
 
         return static function (FrameworkConfig $framework) {
-            $framework
-                ->exceptions(BadRequestHttpException::class)
-                ->log_level('debug');
-
-            $framework
-                ->exceptions(BadRequestHttpException::class)
-                ->status_code(422);
+            $framework->exception(BadRequestHttpException::class)
+                ->logLevel('debug')
+                ->statusCode(422)
             ;
         };
 
```

Comment:
Deprecated directive for version 6.3 is missing. Can you add it?

Reply:
Is that really necessary since the deprecated format was never documented?

Reply:
I agree, it is not necessary